packages/frameworks/src/frameworks.ts

```diff
@@ -855,14 +855,14 @@ export const frameworks = [
         placeholder: '`npm run build` or `rollup -c`',
       },
       devCommand: {
-        value: 'sirv public --single --dev --port $PORT',
+        value: 'rollup -c -w',
       },
       outputDirectory: {
         value: 'public',
       },
     },
     dependency: 'sirv-cli',
```

Comment:
I guess this doesn't make sense either then, since `sirv-cli` doesn't seem to be specific for Svelte

Reply:
Do we need to change line 846 to something else? Its currently matching `sirv-cli` as a dependency

Reply:
I think it would make the most sense to add svelte as dependency there as well, to ensure both are matched

Reply:
Updated the check, I guess we can leave this one for now, since it's only used as a fallback at build time